Big-Spender/readme.md

```diff
@@ -40,16 +40,17 @@ You are a data analyst working with the finance team at Worcestershire Acute Hos
 You are working with Claire and Farnoosh, who are trying to complete a missing report for their boss. They don't just want the answers, they want the queries that will give them the answers. They want to be able to run the queries themselves, so they can do this year's report without your help.
 
 **Claire:** Hey, can you help us out with something? We need to analyze our spending data for 2021 because apparently the report is missing.
+SELECT date FROM spends WHERE date BETWEEN '2021-01-01' AND '2021-12-31';
 
 **You:** I can try. What kind of data are you looking for exactly?
 
 **Farnoosh:** We need to find out the transactions between £30,000 and £31,000. Could you help us write a query for that?
 
 **You:** Absolutely. Here's the SQL query you need:
 
-```sql
 INSERT YOUR QUERY HERE
-```
+
+SELECT amount FROM spends WHERE amount BETWEEN 30000 AND 31000;
```

Comment:
nice use of caps for SQL keywords. what if we also want to see the transactions as well? This would only give us the amount value, which might not be as helpful.